stdlib/stdlib.ml

```diff
@@ -68,15 +83,19 @@ let max x y = if x >= y then x else y
 external ( == ) : 'a -> 'a -> bool = "%eq"
 external ( != ) : 'a -> 'a -> bool = "%noteq"
 
-(* Boolean operations *)
+(* Boolean type and operations *)
+
+type nonrec bool = bool = false | true
 
 external not : bool -> bool = "%boolnot"
 external ( & ) : bool -> bool -> bool = "%sequand"
 external ( && ) : bool -> bool -> bool = "%sequand"
 external ( or ) : bool -> bool -> bool = "%sequor"
 external ( || ) : bool -> bool -> bool = "%sequor"
 
-(* Integer operations *)
+(* Integer type and operations *)
+
+type nonrec int = int
```

Comment:
We already have `Stdlib.Int.t`, so to be coherent with `Stdlib.Array.t`, etc, we might not want to re-expose directly `Stdlib.int` (one counterargument is that some values in `Stdlib` use `int`, but none use `array`).